keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/SecurityGroupModel.kt

```diff
@@ -26,13 +27,15 @@ data class SecurityGroupModel(
   val region: String,
   val vpcId: String?,
   val inboundRules: Set<SecurityGroupRule> = setOf(),
-  val moniker: Moniker
+  val moniker: Moniker,
+  val tags: Set<Tag> = setOf()
 ) {
   data class SecurityGroupRule(
     val protocol: String?,
     val portRanges: List<SecurityGroupRulePortRange>?,
     val securityGroup: SecurityGroupRuleReference?,
-    val range: SecurityGroupRuleCidr?
+    val range: SecurityGroupRuleCidr?,
+    val description: String?
```

Comment:
Required for the Netflix internal implementation to make determinations about the security group rules.